packages/frontend/src/api/useDialogs.tsx

```diff
@@ -158,15 +165,21 @@ export const getViewType = (dialog: InboxItemInput): InboxViewType => {
 
 export const useDialogs = (parties: PartyFieldsFragment[]): UseDialogsOutput => {
   const { organizations } = useOrganizations();
-  const partyURIs = parties.map((party) => party.party);
+  const { selectedParties } = useParties();
+
```

Comment:
You are doing the same operation twice, maybe this could be a separate function and you could write a simple test to ensure it works as intended?